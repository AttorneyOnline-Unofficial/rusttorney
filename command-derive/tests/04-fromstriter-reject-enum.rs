use command_derive::*;

#[derive(FromStrIter)]
enum Handshake {
    One,
    Two,
    Three
}

fn main() {}
