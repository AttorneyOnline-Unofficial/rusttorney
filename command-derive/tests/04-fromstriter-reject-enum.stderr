error: expected `struct`
 --> $DIR/04-fromstriter-reject-enum.rs:4:1
  |
4 | enum Handshake {
  | ^^^^
